### DISCUSSION TITLE
BeamCore, suggestion by Kilo

### DISCUSSION THREAD

**Discussion by ELPatitoLoco** [TAServers/issues#11] [Addon Suggestions]
https://steamcommunity.com/sharedfiles/filedetails/?id=392679625 (0.016 MB)
Small E2 Core that allows beams to be created.
If this is bad, just lemme know and i'll just make a better core.
Kilo

**Comment by Derpius** [comment]
Hasn't had an update in ages so may not work with the E2 breaking changes:

![image](https://github.com/TAServers/issues/assets/49565664/2dcf2223-f6e3-4022-9e29-1e1463b6d91f)
![image](https://github.com/TAServers/issues/assets/49565664/f70a587b-29ba-4d7d-b23d-3ea820c2de75)


I haven't actually checked if there's any issues yet though

**Reply by Kilometres** [reply]
I tried it out before the suggestion and didn't experience any bugs, but if what vurv said does become a problem I'll just fork it so it returns the correct types.

**Comment by Derpius** [comment]
Will add now, but please report any issues/exploits (either in Jira or at https://github.com/TAServers/issues/issues actually)
